### PR TITLE
Implement reliable calculation of parameters for differential privacy

### PR DESCRIPTION
**Pull request by RaffaelBild**

### CODE REVIEW

File: src/main/org/deidentifier/arx/criteria/EDDifferentialPrivacy.java [hunk 1]
```diff
@@ -98,9 +91,9 @@ public EDDifferentialPrivacy(double epsilon, double delta,
         this.epsilon = epsilon;
```
Comment (prasser) [inline src/main/org/deidentifier/arx/criteria/EDDifferentialPrivacy.java]: Check comment "Note: ...". Also check, if changing the constructor breaks anything.

File: src/main/org/deidentifier/arx/dp/ParameterCalculation.java [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+     * @param k
+     * @param epsilon
+     * @param beta
+     * @param gamma
+     * @return
+     * @throws IntervalArithmeticException 
+     */
+    private IntervalDouble calculateDelta(int k, IntervalDouble epsilon, IntervalDouble beta, IntervalDouble gamma) throws IntervalArithmeticException {
+        
+        // double gamma = calculateGamma(epsilon, beta);
```
Comment (prasser) [inline src/main/org/deidentifier/arx/dp/ParameterCalculation.java]: Comments.

File: src/main/org/deidentifier/arx/reliability/IntervalDouble.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+ * 
+ * @author Fabian Prasser
+ */
+public class IntervalDouble implements Serializable {
+
+    /** SVUID */
+    private static final long serialVersionUID = 6012504736748464073L;
+
+    /** Lower */
+    final double              lower;
```
Comment (prasser) [inline src/main/org/deidentifier/arx/reliability/IntervalDouble.java]: Visibility vs getter.

**Comment by RaffaelBild** [comment]
Thank you for the feedback. I have performed the suggested changes.

**Pull request merged**